workers/worker-performance.worker.js

```diff
@@ -0,0 +1,128 @@
+"use strict";
+
+// Tests that most of the functionality of the window.performance object is available in web workers.
```

Comment:
Is this needed? Aren't most `performance` tests written to run on both a page and a worker?

Reply:
For example this test seems to duplicate some of what `user-timing/mark*.any.js`, `user-timing/clear*.any.js`, and `user-timing/measure*.any.js` cover? Is it expected to have duplication of testing across different features (here having a test in `workers` duplicating `user-timing`, `resource-timing`, etc). I'm fine with it, I just don't really know WPT policies. I do see that none of the resource-timing tests test in workers, so there is certainly some merit for this test.

Reply:
@plehegar any guidance here? I can see arguments both ways.. duplication is not good (tm), but at the same time, some redundancy is OK in my books -- e.g. if someone is running worker test suite only, this would help signal potential issues with perf primitives.

Reply:
Some redundancy is also ok in my books. This test can indeed encourage support of RT in workers.